Cabal/Cabal.cabal

```diff
@@ -1,5 +1,5 @@
 name:          Cabal
-version:       2.2.0.1
+version:       2.2.0.2
```

Comment:
why bump `Cabal`?

Reply:
Oh, I see, added symbol. That should require third digit bump, patch (4th) level isn't enough.